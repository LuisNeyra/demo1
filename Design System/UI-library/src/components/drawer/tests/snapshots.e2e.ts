import { snapIt } from '../../../test';

const component = 'drawer';

describe(component, () => {
  const snap = snapIt(component);

  describe('renders', () => {
    snap('renders correctly with no props', '<blaze-drawer>default</blaze-drawer>');
    snap('renders as open', '<blaze-drawer open>default</blaze-drawer>');
    snap('renders dismissible overlay', '<blaze-drawer open dismissible>default</blaze-drawer>');
  });
});
