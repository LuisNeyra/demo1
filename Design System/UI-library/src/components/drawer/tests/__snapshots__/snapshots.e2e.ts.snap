// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`drawer renders renders as open 1`] = `
"<blaze-drawer open=\\"\\" class=\\"hydrated\\"><!----><div aria-hidden=\\"true\\" class=\\"c-overlay c-overlay--fullpage c-overlay--visible\\"></div><aside aria-expanded=\\"async isOpen() {
    return this._isOpen;
  }\\" class=\\"o-drawer o-drawer--bottom o-drawer--visible\\">default</aside></blaze-drawer>"
`;

exports[`drawer renders renders correctly with no props 1`] = `
"<blaze-drawer class=\\"hydrated\\"><!----><div aria-hidden=\\"true\\" class=\\"c-overlay c-overlay--fullpage\\"></div><aside aria-expanded=\\"async isOpen() {
    return this._isOpen;
  }\\" class=\\"o-drawer o-drawer--bottom\\">default</aside></blaze-drawer>"
`;

exports[`drawer renders renders dismissible overlay 1`] = `
"<blaze-drawer open=\\"\\" dismissible=\\"\\" class=\\"hydrated\\"><!----><div aria-hidden=\\"true\\" class=\\"c-overlay c-overlay--fullpage c-overlay--visible\\"></div><aside aria-expanded=\\"async isOpen() {
    return this._isOpen;
  }\\" class=\\"o-drawer o-drawer--bottom o-drawer--visible\\">default</aside></blaze-drawer>"
`;
